### ISSUE TITLE
NodeJS / GraphQL resolvers test does not show code coverage

### ISSUE THREAD

**Issue by DHFW** [wallabyjs/public#2214]
### Issue description or question
Is it possible to show inline code coverage for resolvers in a NodeJS, Apollo-server configuration with inline code coverage in VS Code? The inline code coverage works for the other files, like helpers or services but I would like to see the code coverage of the resolvers.

Currently, is shows only white boxes for each statement, except the `export`ed const resolver, that is green. Also the code coverage report is giving me info that only that first statement is being counted as covered code...

Each GraphQL resolver test is executed with a request-promise-native client which is actually calling the real endpoint. So it is like an e2e test...

### Wallaby diagnostics report

(I Removed superfluous and sensitive info from the report)

```
{ editorVersion: '1.38.1',
  pluginVersion: '1.0.143',
  editorType: 'VSCode',
  osVersion: 'darwin 18.7.0',
  nodeVersion: 'v10.14.1',
  coreVersion: '1.0.751',
  config:
   { files:
      [ { pattern: 'src/**/*.ts', load: false, ignore: false, trigger: true, instrument: true, order: 1 },
        { pattern: 'src/**/*.gql', load: false, ignore: false, trigger: true, instrument: true, order: 2 },
        { pattern: 'src/**/*.md', load: false, ignore: false, trigger: true, instrument: true, order: 3 },
        { pattern: 'src/**/*.txt', load: false, ignore: false, trigger: true, instrument: true, order: 4 },
        { pattern: 'src/**/*.test.ts', ignore: true, trigger: true, load: true, instrument: true },
        { pattern: 'src/**/*.test.ts.snap', ignore: false, instrument: false, trigger: true, load: true, order: 5 },
        { pattern: 'package.json', ignore: false, instrument: false, trigger: true, load: true, order: 6 } ],
     filesWithNoCoverageCalculated: [ 'src/**/*mock.ts' ],
     tests: [ { pattern: 'src/**/*.test.ts', ignore: false, trigger: true, load: true, test: true, order: 7 } ],
     env: { type: 'node', runner: 'node', params: {}, viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
     compilers: { '**/*.?(lit)coffee?(.md)': [Function] },
     testFramework: { version: 'jest@0.4.3', configurator: 'jest@0.4.3', reporter: 'jest@0.4.3', starter: 'jest@0.4.3' },
     debug: false,
     runAllTestsInAffectedTestFile: false,
     preprocessors: { 'package.json': [Function] },
     maxConsoleMessagesPerTest: 100,
     autoConsoleLog: true,
     delays: { run: 0, edit: 100, update: 0 },
     workers: { initial: 0, regular: 0, recycle: false },
     teardown: undefined,
     hints:
      { ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
        ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
        commentAutoLog: '?',
        testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' } },
     automaticTestFileSelection: true,
     runSelectedTestsOnly: false,
     extensions: {},
     reportUnhandledPromises: true,
     slowTestThreshold: 75,
     lowCoverageThreshold: 80,
     loose: true,
     configCode:
      'process.env.NODE_ENV=\'test\';\nconst settings = require(\'./src/utils/settings.js\');\n\n\nmodule.exports = function (wallabyModule) {\n  return {\n    files: [\n      {pattern: \'src/**/*.ts\', load: false},\n      {pattern: \'src/**/*.gql\', load: false},\n      {pattern: \'src/**/*.md\', load: false},\n      {pattern: \'src/**/*.txt\', load: false},\n      {pattern: \'src/**/*.test.ts\', ignore: true},\n    ],\n    filesWithNoCoverageCalculated: [\'src/**/*mock.ts\'],\n    tests: [\'src/**/*.test.ts\'],\n    env: {\n      type: \'node\',\n      runner: \'node\'\n    },\n    compilers: {\n      \'**/*.ts\': wallabyModule.compilers.typeScript({\n        // module: \'commonjs\',\n      })\n    },\n    testFramework: \'jest\',\n    bootstrap: function (wallaby) {\n      const jestConfig = require(wallaby.localProjectDir + \'/jest.config.js\');\n      wallaby.testFramework.configure(jestConfig);\n    },\n    debug: false\n  };\n};\n' },
  packageJSON:
   { dependencies:
      { '@elastic/elasticsearch': '7.3.0',
        '@gql2ts/from-schema': '1.10.1',
        'apollo-server': '^2.3.1',
        'apollo-server-errors': '2.2.0',
        'apollo-server-express': '^2.3.1',
        axios: '0.19.0',
        'babel-plugin-import-graphql': '2.7.0',
        'big-number': '1.0.0',
        bl: '1.0.2',
        'body-parser': '1.19.0',
        'cls-hooked': '4.2.2',
        concurrently: '3.6.0',
        'cookie-parser': '1.4.4',
        cors: '2.8.5',
        'date-fns': '2.0.0-alpha.16',
        'dev-null': '^0.1.1',
        dotenv: '6.0.0',
        errorhandler: '1.5.1',
        express: '4.17.1',
        'express-session': '1.16.2',
        'form-data': '2.5.1',
        glob: '7.1.4',
        graphql: '14.5.6',
        'graphql-resolvers': '0.3.3',
        'graphql-tools': '4.0.5',
        'guid-typescript': '1.0.9',
        knex: '0.16.5',
        lodash: '4.17.15',
        lowdb: '1.0.0',
        marked: '^0.7.0',
        'merge-graphql-schemas': '1.7.0',
        mkdirp: '0.5.1',
        morgan: '1.9.1',
        mssql: '5.1.0',
        nodemailer: '^5.1.1',
        objection: '1.6.8',
        'objection-graphql': '0.4.2',
        'promise-all': '1.0.0',
        request: '2.88.0',
        'request-promise': '4.2.4',
        'request-promise-native': '1.0.7',
        shortid: '2.2.15',
        'tough-cookie': '2.5.0',
        winston: '3.2.1',
        'winston-elasticsearch': '0.7.13' },
     devDependencies:
      { '@babel/cli': '7.1.5',
        '@babel/core': '7.1.6',
        '@babel/node': '7.0.0',
        '@babel/plugin-proposal-class-properties': '7.1.0',
        '@babel/plugin-proposal-decorators': '7.1.6',
        '@babel/plugin-proposal-object-rest-spread': '7.0.0',
        '@babel/preset-env': '7.1.6',
        '@babel/preset-typescript': '7.1.0',
        '@types/bl': '2.1.0',
        '@types/body-parser': '1.17.0',
        '@types/cls-hooked': '4.3.0',
        '@types/cookie-parser': '1.4.1',
        '@types/dotenv': '6.1.0',
        '@types/errorhandler': '0.0.32',
        '@types/express': '4.16.0',
        '@types/express-session': '1.15.11',
        '@types/glob': '7.1.1',
        '@types/graphql': '14.5.0',
        '@types/jasmine': '3.4.0',
        '@types/jest': '24.0.18',
        '@types/knex': '0.15.1',
        '@types/lodash': '4.14.118',
        '@types/lowdb': '1.0.6',
        '@types/marked': '^0.6.3',
        '@types/mkdirp': '0.5.2',
        '@types/morgan': '1.7.35',
        '@types/node': '10.12.11',
        '@types/request-promise': '4.1.42',
        '@types/request-promise-native': '1.0.15',
        '@types/shortid': '0.0.29',
        chai: '4.2.0',
        connect: '3.6.6',
        'cross-conf-env': '^1.1.2',
        jest: '24.9.0',
        'ng-apimock': '1.4.8',
        nodemon: '1.19.2',
        prettier: '1.15.3',
        rimraf: '2.6.2',
        'ts-jest': '24.1.0',
        'ts-node': '7.0.1',
        tslint: '5.20.0',
        'tslint-config-airbnb': '5.11.2',
        'tslint-config-prettier': '1.18.0',
        'tslint-consistent-codestyle': '1.15.1',
        'tslint-eslint-rules': '5.4.0',
        typescript: '3.6.3' } },
  fs: { numberOfFiles: 262 },
  debug:
   [ '2019-09-20T15:38:44.112Z project Wallaby Node version: v10.14.1\n',
     '2019-09-20T15:38:44.112Z project Wallaby config: <homeDir>/xxx/wallaby.js\n',
     '2019-09-20T15:38:44.777Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/projects/a1f6f0f8b7f6a781\n',
     '2019-09-20T15:38:44.791Z uiService Listening port 51235\n',
     '2019-09-20T15:38:44.798Z project package.json file change detected, invalidating local cache\n',
     '2019-09-20T15:38:44.855Z workers Parallelism for initial run: 10, for regular run: 5\n',
     '2019-09-20T15:38:44.855Z workers Starting run worker instance #0\n',
     '2019-09-20T15:38:44.856Z workers Web server is listening at 51843\n',
     '2019-09-20T15:38:44.856Z project File cache requires some updates, waiting required files from IDE\n',
     '2019-09-20T15:38:45.431Z project Starting processor pool (if not yet started)\n',
     '2019-09-20T15:38:45.509Z uiService UI client connected\n',
     '2019-09-20T15:38:45.510Z uiService Outgoing message ui:handshake\n',
     '2019-09-20T15:38:45.518Z uiService Incoming message ui:start\n',
     '2019-09-20T15:38:45.518Z uiService Outgoing message ui:summary\n',
     '2019-09-20T15:38:45.521Z uiService Outgoing message ui:files\n',
     '2019-09-20T15:38:45.535Z uiService Incoming message ui:closedInEditor\n',
     '2019-09-20T15:38:45.536Z uiService Incoming message ui:codeRequested\n',
     '2019-09-20T15:38:45.537Z uiService Outgoing message ui:code\n',
     '2019-09-20T15:38:45.539Z uiService Incoming message ui:openedInEditor\n',
     '2019-09-20T15:38:45.548Z uiService Outgoing message ui:busy\n',
     '2019-09-20T15:38:46.032Z workers Started run worker instance (delayed) #5\n',
     '2019-09-20T15:38:46.033Z workers Started run worker instance (delayed) #1\n',
     '2019-09-20T15:38:46.033Z workers Started run worker instance (delayed) #8\n',
     '2019-09-20T15:38:46.064Z uiService Outgoing message ui:busy\n',
     '2019-09-20T15:38:46.084Z workers Started run worker instance (delayed) #7\n',
     '2019-09-20T15:38:46.103Z workers Started run worker instance (delayed) #9\n',
     '2019-09-20T15:38:46.115Z uiService Outgoing message ui:busy\n',
     '2019-09-20T15:38:46.147Z workers Started run worker instance (delayed) #4\n',
     '2019-09-20T15:38:46.150Z workers Started run worker instance (delayed) #2\n',
     '2019-09-20T15:38:46.151Z workers Started run worker instance (delayed) #0\n',
     '2019-09-20T15:38:46.155Z workers Started run worker instance (delayed) #6\n',
     '2019-09-20T15:38:46.158Z workers Started run worker instance (delayed) #3\n',
     '2019-09-20T15:38:46.166Z uiService Outgoing message ui:busy\n',
     '2019-09-20T15:38:47.036Z project Starting processor pool (if not yet started)\n',
     '2019-09-20T15:38:47.037Z project Starting processor pool (if not yet started)\n',
     '2019-09-20T15:38:47.037Z project Starting processor pool (if not yet started)\n',
     '2019-09-20T15:38:47.041Z project Stopping process pool\n',
     '2019-09-20T15:38:47.042Z uiService Outgoing message ui:busy\n',
     '2019-09-20T15:38:47.042Z project Running postprocessor\n',
     '2019-09-20T15:38:47.100Z postprocessor New TypeScript language service is required\n',
     '2019-09-20T15:38:50.772Z project Re-attaching document block comment\n',
     '2019-09-20T15:38:50.938Z project Postprocessor execution finished\n',
     '2019-09-20T15:38:50.939Z project Test run started; run priority: 3\n',
     '2019-09-20T15:38:50.939Z project Running all tests\n',
     '2019-09-20T15:38:50.954Z workers Starting test run, priority: 3\n',
     '2019-09-20T15:38:50.955Z workers Distributing tests between 10 workers\n',
     '2019-09-20T15:38:50.955Z workers Running tests in parallel\n',
     '2019-09-20T15:39:00.978Z project Processed loading sequences\n',
     '2019-09-20T15:39:00.981Z project Processed executed tests\n',
     '2019-09-20T15:39:01.000Z project Processed code coverage\n',
     '2019-09-20T15:39:01.079Z project Test run result processed and sent to IDE\n',
     '2019-09-20T15:39:01.079Z uiService Outgoing message ui:summary\n',
     '2019-09-20T15:39:01.080Z uiService Outgoing message ui:coverageChanged\n',
     '2019-09-20T15:39:01.081Z uiService Outgoing message ui:files\n',
     '2019-09-20T15:39:01.085Z uiService Incoming message ui:codeRequested\n',
     '2019-09-20T15:39:01.086Z uiService Outgoing message ui:code\n' ] }
```

**Comment by smcenlly** [comment]
I can see that you are using Jest v24+. Do your tests run from the command line using `npx jest`? If so, it might be worth checkout out our new [automatic configuration feature](https://wallabyjs.com/docs/intro/config.html#automatic-configuration) (available for Jest v24+ and Angular CLI v8.2.0+).

Coverage is reported a little bit differently with automatic configuration (the same as it is in Jest) so you may find that this just works for you.

If that doesn't work for you, are you able to send us a sample repo (we're not familiar with NodeJS, Apollo-server configuration, and Graphql resolvers)?

**Comment by DHFW** [comment]
@smcenlly Thank you for your quick reply. I used the automatic configuration feature, and... now it works just fine! Both for console.log's that I before wasn't able to see in the Wallaby.js Tests output window (they were visible in the Wallaby.js Console window) and the code coverage!

I'm very happy for your suggestion (and I love Wallaby.js!). Happy user again!
